Algorithm/QCAlgorithm.cs

```diff
@@ -3398,12 +3398,13 @@ public List<Fundamental> Fundamentals(List<Symbol> symbols)
             return symbols.Select(symbol => Fundamentals(symbol)).ToList();
         }
 
+
         /// <summary>
-        /// Get the option chain for the specified symbol at the current time (<see cref="Time"/>)
```

Comment:
Same here, don't think we need to replace "option chain" which the class, the return type of the method gives the information needed for that